index.html

```diff
@@ -4,10 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello World</title>
-    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/e52d74ec85.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,700;1,300;1,500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <title>Jiachun's Portfolio</title>
 </head>
 <body>
-    <h1>Hello Microverse!</h1>
+    <header>
+        <p><a href="#">My Logo</a></p>
+        <nav class="burger-icon">
+            <i class="fa-solid fa-bars"></i>
+        </nav>
+    </header>
+    <section class="hero">
+        <div class="container hero-text">
+            <h1>I'm Jiachun, Glad to see you</h1>
+            <p>I'm a software developer, I can help you build a product, 
+            feature or Website. Look trough some of my work and experience, 
+            if you like what you see and have a project you need coded, don't
+            hesitate to contact me. 
+            </p>
+            <span>LET'S CONNECT</span>
+            <ul class="social-media">
+                <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/jiachun-lily-xiang-1b197213a/"><img src="./img/linkedin.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/M.png" alt="M icon"></a></li>
+                <li><a href="https://github.com/mpa-mxiang"><img src="./img/git.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/hand.png" alt="hand icon"></a></li>
```

Comment:
- [Optional] Your social media icons are too big compared with the Figma design. Please address this for a better layout.
![Screenshot 2023-01-13 085421](https://user-images.githubusercontent.com/69077061/212247672-af391577-44ae-4ec2-ace6-abf03c074cee.png)